env/scratch/fluentbit.yaml

```diff
@@ -306,8 +337,10 @@ data:
         flush_timeout 1000
         # rules |   state name  | regex pattern         | next state
         # ------|---------------|-----------------------------------
-        rule      "start_state"   "/^\[.*\].*/"                "cont"
-        rule      "cont"          "/^[^\[].*/"           "cont"
+        rule      "start_state"   "/^\b\d{1,3}\.\d{1,3}\.\d{1,3}\.\d{1,3}\b|^\[.*\]/"           "cont"
+        rule      "cont"          "/^[\s].*|^\b(Traceback|Exception)\b.*/"  "cont"
```

Comment:
That seems good enough, but I wonder if we had cases where multi line continuation was required and there were no whitespace characters in the beginning? 🤔 Let's test this thoroughly once it hits staging. Tell me and  I can help with the testing.

Reply:
Yeah and that's where the problem occurred... if there's no whitespace, the regex picks up all new log entries... I tried doing a pattern "not" like IP address or date, but couldn't get it working. If this proves to be insufficient, maybe we can link up and try and debug my other pattern.